github/plugin.py

```diff
@@ -735,3 +786,71 @@ def invite_user_to_gh_org_team(self, github_id, team_id):
 
         if response.status_code != 200:
             raise ValueError('GitHub Problem: Adding to team, status code: {}'.format(response.status_code))
+
+    def list_pull_requests(self, data, user_data):
+        """
+        Sets the default branch of a repo.
+
+        Command is as follows: !listprs <organization> <repo> <state>
+        :param data:
+        :return:
+        """
+        try:
+            parser = argparse.ArgumentParser()
+            parser.add_argument('org', type=str)
+            parser.add_argument('repo', type=str)
+            parser.add_argument('state', type=str)
+
+            args, unknown = parser.parse_known_args(args=preformat_args(data["text"]))
+            if len(unknown) > 0:
+                raise SystemExit()
+
+            args = vars(args)
+
+            # Check that we can use this org:
+            real_org = self.org_lookup[args["org"]][0]
+            reponame = extract_repo_name(args["repo"])
+            state = args["state"]
```

Comment:
May want to add some logic for verifying the state.  Per GitHub's documentation, it's either `open`, `closed`, or `all`.  

I do something similar on: https://github.com/Netflix/hubcommander/blob/develop/github/plugin.py#L278-L280. In that case, I'm adding those to the `dict`, but that could be done inline as well.